### PR TITLE
Docs brand designed and user manual added

### PR DESCRIPTION
**Pull request by SirZesso**
- SmashGrade Logos added 
- Theme color changed to the hftm design 
- user manual part created (first version only with chapters!) 
- software architecture chapters moved to a sub-folder

### CODE REVIEW

**Comment by joshua-lehmann** [comment]
@SirZesso sieht gut aus danke für die Design Anpassungen. Anstelle von Pink könntest du auch Custom colors verwenden, ist aber nicht nötigt. Die Doku war ursprünglich für die technischen Dinge gedacht, weshalb sie auch komplett Englisch ist. Dein User Manual ist aber auf Deutsch, für mich ist das okay, sprich es bitte aber noch mit @Saverbox und @stafel ab was sie dazu meinen.

Ausserdem behebe bitte noch dieses warning:
image

File: docs/01_smash_grade_app
Comment (joshua-lehmann) [inline docs/01_smash_grade_app:1]: Sollte wahrscheinlich ein .md file sein?

File: docs/index.md
Comment (joshua-lehmann) [inline docs/index.md:1]: Durch dein Löschen des index.md gibt es nun keine "homepage" mehr und es wird ein 404 angezeigt auf der Startseite. Bitte entweder eines deiner Dokumente als index.md benennen (die gewünschte Einstiegsseite) oder eine neue Übersichtsseite erstellen

**Review by joshua-lehmann — CHANGES_REQUESTED** [review]

**Review by stafel — CHANGES_REQUESTED** [review]
- user manuals and guides should be tied together with the version they are referencing else they won't match. Add them to the frontend repo and keep them in sync or make a separate repo?
- this repo is for the architecture documentation in the style of arc42
- language english only
- each ar42 section should have a separate folder to allow multiple edits on the same branch with minimal merge conflict potential
- we probably should document all this in an ADR

File: docs/01_smash_grade_app
Comment (SirZesso) [reply docs/01_smash_grade_app:1]: fixed: [422a30e](https://github.com/SmashGrade/documentation/pull/22/commits/422a30e7b88aa0efde085ba2030057c915920036)

**Comment by SirZesso** [comment]
Thank you @joshua-lehmann and @stafel for your comments :) 

- Removed all the new stuff 
- Only kept changes to the style & logo

File: docs/index.md
Comment (SirZesso) [reply docs/index.md:1]: fixed: [234a0e3](https://github.com/SmashGrade/documentation/pull/22/commits/234a0e3699df02beda0b55f8d0a279ca135b4b70)

File: docs/index.md [hunk 1]
```diff
@@ -1,17 +1,14 @@
-# Welcome to MkDocs
+# Welcome message
+Welcome, fellow architects, to the fascinating world of 'Smash Grade'! 
 
-For full documentation visit [mkdocs.org](https://www.mkdocs.org) and 
+Imagine a Swiss timepiece meticulously designed with intricate precision, our application is no different — a harmonious blend of technology and innovation.
```
Comment (stafel) [inline docs/index.md:4]: Are you sure prose about watchmaking sends the right welcome message for developers checking out technical documentation of a grading app?
It probably has more effect as an epigram in the user manual.

**Review by stafel — CHANGES_REQUESTED** [review]

**Comment by joshua-lehmann** [comment]
> * user manuals and guides should be tied together with the version they are referencing else they won't match. Add them to the frontend repo and keep them in sync or make a separate repo?
> * this repo is for the architecture documentation in the style of arc42
> * language english only
> * each ar42 section should have a separate folder to allow multiple edits on the same branch with minimal merge conflict potential
> * we probably should document all this in an ADR

@stafel Thanks for your additional points they are very valid. About the user manual, I get your point and agree. However I don't think the manual should be inside the frontend repo. Since also backend changes (bug fixes, new feature, new validation, different api etc.) can affect the functionality and might require changes in the manual. I think as soon as we have a tight integration with the backend, we anyways need to agree on a versioning strategy/coupling between frontend and backend, right?

Long story short, a user manual is a good idea and publish it with MKDOCS as well since it[ has already a versioning system](https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning/) included that can be enabled and makes it available to the web. However I am not sure if it makes sense to have a separate repo and doc site just for the manual? Then we will have 3 documentations already (word doc, architecture doc site and user manual site). If we have a complete separate folder inside the mkdocs for architecture and manual, you think it would still bother? This is something we should quickly discuss in a chat/call with all involved people, what would be the best and simplest way for all of us to create a user manual and make sure it is up to date with the current app version.

**Comment by stafel** [comment]
> Long story short, a user manual is a good idea and publish it with MKDOCS as well since it[ has already a versioning system](https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning/) included that can be enabled and makes it available to the web. However I am not sure if it makes sense to have a separate repo and doc site just for the manual? Then we will have 3 documentations already (word doc, architecture doc site and user manual site). If we have a complete separate folder inside the mkdocs for architecture and manual, you think it would still bother? This is something we should quickly discuss in a chat/call with all involved people, what would be the best and simplest way for all of us to create a user manual and make sure it is up to date with the current app version.

I fully agree with using mkdocs for it. The versioning addon for it looks promising.

My initial idea was a user documentation for each part in the corresponding repository to keep it synced with the code itself and reference the other side via version tags. But this fragments the docu even further.

**Review by stafel — APPROVED** [review]

File: docs/index.md (hunk 1 above)
Comment (SirZesso) [reply docs/index.md:4]: Ja stimmt, das können wir dann noch auslöschen 👍🏻

**Review by joshua-lehmann — APPROVED** [review]

**Pull request merged**